contribute/design/README.md

```diff
@@ -31,16 +31,72 @@ Storage Device Management Plane can be viewed as an Inventory Management tool, t
 
 # Source Code and Dependencies 
 
-| Component | Repo Link | License | Dependency and Notes
-|---|---|---|---|
-| Storage Management | https://github.com/openebs/maya | Apache 2.0  | Contains all the OpenEBS Storage Management components that help with managing multiple data engines. Dependencies are in: https://github.com/openebs/maya/blob/master/Gopkg.lock
-| Storage Management - Dynamic Provisioners | https://github.com/openebs/external-storage | Apache 2.0  | Contains OpenEBS extensions for Kubernetes External Dynamic Provisioners. This will be replaced with CSI drivers under the above repo. 
-| Data Engine - Jiva | https://github.com/openebs/jiva | Apache 2.0  | The first data engine supported by OpenEBS. Forked out from Rancher Longhorn engine. Dependencies are in: https://github.com/openebs/jiva/blob/master/vendor.conf
-| Data Engine - cStor Replica | https://github.com/openebs/libcstor | Apache 2.0  | The second data engine supported by OpenEBS. cStor Data Engine makes use of (uZFS - userspace ZFS, aka cStor)[https://github.com/openebs/cstor].
-| Data Engine - cStor Target | https://github.com/openebs/istgt | Apache 2.0  | Part of the cStor data engine that implements the iSCSI Target. 
-| Storage Device Management | https://github.com/openebs/node-disk-manager | Apache 2.0  | Contains Kubernetes Device Inventory Management functionality. Dependencies are in https://github.com/openebs/node-disk-manager/blob/master/Gopkg.lock
-| OpenEBS Documentation | https://github.com/openebs/openebs-docs | Apache 2.0  | Uses Docusaurus. Dependencies are in https://github.com/openebs/openebs-docs/blob/staging/website/package.json
-| OpenEBS Examples | https://github.com/openebs/openebs | Apache 2.0  | Wrapper repo for examples and project management.|
+OpenEBS source code is spread across multiple repositories, organized either by the storage engine or management layer. This section describes the various actively maintained repositories. 
+
+## Common or Generic 
+
+- [openebs/openebs](https://github.com/openebs/openebs) is OpenEBS meta repository that contains design documents, project management, community and contributor documents, deployment and workload examples. 
+- [openebs/openebs-docs](https://github.com/openebs/openebs-docs) contains the source code for OpenEBS Documentation portal (https://docs.openebs.io) implemented using Docusaurus framework and other libraries listed in [package.json](https://github.com/openebs/openebs-docs/blob/staging/website/package.json)
+- [openebs/website](https://github.com/openebs/website) contains the source code for OpenEBS portal (https://openebs.io) implemented using Gatsby framework and other libraries listed in [package.json](https://github.com/openebs/website/blob/refactor-to-ghost-and-gatsby/package.json)
+- [openebs/charts](https://github.com/openebs/charts) contains the helm chart source code for OpenEBS and also hosts a gh-pages website for install artifacts and helm packages. 
```

Comment:
```suggestion
- [openebs/openebs](https://github.com/openebs/openebs) is OpenEBS meta repository that contains design documents, project management, community and contributor documents, deployment and workload examples. 
- [openebs/openebs-docs](https://github.com/openebs/openebs-docs) contains the source code for OpenEBS Documentation portal (https://docs.openebs.io) implemented using Docusaurus framework and other libraries listed in [package.json](https://github.com/openebs/openebs-docs/blob/staging/website/package.json).
- [openebs/website](https://github.com/openebs/website) contains the source code for OpenEBS portal (https://openebs.io) implemented using Gatsby framework and other libraries listed in [package.json](https://github.com/openebs/website/blob/refactor-to-ghost-and-gatsby/package.json).
- [openebs/charts](https://github.com/openebs/charts) contains the helm chart source code for OpenEBS and also hosts a gh-pages website for install artifacts and helm packages. 
```

Reply:
We should be consistent with line endings. Either all of the bullet points have line endings, or none should have line endings.